x11-libs/libxtst/libxtst-1.2.3.recipe

```diff
@@ -5,62 +5,79 @@ HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
 COPYRIGHT="1990, 1991 UniSoft Group Limited
 	1992, 1993, 1995, 1998 The Open Group"
 LICENSE="MIT (no promotion)"
-REVISION="1"
-SOURCE_URI="https://www.x.org/archive/individual/lib/libXtst-$portVersion.tar.gz"
-CHECKSUM_SHA256="a0c83acce02d4923018c744662cb28eb0dbbc33b4adc027726879ccf68fbc2c2"
+REVISION="2"
+SOURCE_URI="https://www.x.org/archive/individual/lib/libXtst-$portVersion.tar.bz2"
+CHECKSUM_SHA256="4655498a1b8e844e3d6f21f3b2c4e2b571effb5fd83199d428a6ba7ea4bf5204"
 SOURCE_DIR="libXtst-$portVersion"
 
-ARCHITECTURES="?x86_gcc2 x86 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+libVersion=6.1.0
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
 	libxtst$secondaryArchSuffix = $portVersion
-	lib:libxtst$secondaryArchSuffix = 6.1.0 compat >= 6
+	lib:libxtst$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libx11$secondaryArchSuffix
+	lib:libxau$secondaryArchSuffix
+	lib:libxcb$secondaryArchSuffix
+	lib:libxdmcp$secondaryArchSuffix
 	lib:libxext$secondaryArchSuffix
-	lib:libX11$secondaryArchSuffix
 	lib:libxi$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
 	libxtst${secondaryArchSuffix}_devel = $portVersion
-	devel:libxtst$secondaryArchSuffix = 6.1.0 compat >= 6
+	devel:libxtst$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES_devel="
 	libxtst$secondaryArchSuffix == $portVersion base
 	"
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:fixesproto$secondaryArchSuffix
-	devel:inputproto$secondaryArchSuffix
-	devel:kbproto$secondaryArchSuffix
-	devel:libpthread_stubs$secondaryArchSuffix
-	devel:libX11$secondaryArchSuffix
+	docbook_xml_dtd
+	docbook_xsl_stylesheets
+	devel:fixesproto
+	devel:inputproto
+	devel:kbproto
+	devel:libpthread_stubs
+	devel:libx11$secondaryArchSuffix
+	devel:libxau$secondaryArchSuffix
+	devel:libxcb$secondaryArchSuffix
+	devel:libxdmcp$secondaryArchSuffix
 	devel:libxext$secondaryArchSuffix
 	devel:libxfixes$secondaryArchSuffix
 	devel:libxi$secondaryArchSuffix
-	devel:recordproto$secondaryArchSuffix
-	devel:xextproto$secondaryArchSuffix
+	devel:recordproto
+	devel:util_macros
+	devel:xextproto
+	devel:xorg_sgml_doctools
 	"
 BUILD_PREREQUIRES="
-	devel:util_macros$secondaryArchSuffix
-	cmd:autoconf
-	cmd:libtool
 	cmd:aclocal
-	cmd:make
+	cmd:autoconf
+	cmd:awk
 	cmd:gcc$secondaryArchSuffix
-	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
+	cmd:xmlto
+	cmd:xsltproc
 	"
 
+defineDebugInfoPackage libxtst$secondaryArchSuffix \
+	$libDir/libXtst.so.$libVersion
+
 BUILD()
 {
-	libtoolize --force --copy --install
-	autoreconf -i
+	autoreconf -vfi
```

Comment:
I don't think this is necessary.

Reply:
I think there are many cases these can be dropped ...